main.js

```diff
@@ -26,22 +26,30 @@ app.on('window-all-closed', () => {
     // nothing
 });
 
+function getData(url) {
+	return new Promise((resolve, reject) => {
+		fetch(url)
+			.then(res => res.json())
+			.then(data => resolve(data))
+			.catch(err => reject(err))
+	})
+}
+
 function fetchData() {
-    fetch('https://world-cup-json.herokuapp.com/matches/today')
-        .then(resp => resp.json())
-        .then(json => {
-            setMenu(json);
-        }).catch(function(err) {
-            console.error(err);
-        });
+	Promise.all([
+		getData('https://world-cup-json.herokuapp.com/matches/today'),
+		getData('https://world-cup-json.herokuapp.com/matches/tomorrow')
```

Comment:
We would only really need to do the call for tomorrow's matches once, not every minute.